SD_Card_Driver/Core/Inc/LOS_D_sd.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef LOS_D_SD_H_
+#define LOS_D_SD_H_
+
+#include "fatfs.h"
+#include "string.h"
+#include "LOS_D_storage_device.hpp"
+
+class SDCard : StorageDevice {

```

Comment:
Did this compile? For inheritance, don't you need to specify the access specifier? such as public, private, or protected... usually we do public.

Reply:
It did compile, so I guess it is public inheritance by default? I'll add a public specifier in the next commit.